### PR TITLE
Adding a quick tour section to Database Basics

### PR DESCRIPTION
**Pull request by lorenzo**
Following the pattern that other documents have for exposing the most important aspects of the chapter contents

### CODE REVIEW

File: en/orm/database-basics.rst [hunk 1]
```diff
@@ -1,9 +1,100 @@
...
-reflection/generation, a flexible type system and more.
+The CakePHP database access layer abstracts and provides help with most aspects
+of dealing with relational database such as keeping connections to the server,
+building queries, preventing SQL injections, inspecting and altering schemas,
+and with debugging and profiling queries sent to the database.
+
+Quick Tour
+==========
+
+The functions described in this chapter relate to what is possible to do with
```
Comment (ravage84) [inline en/orm/database-basics.rst]: I would omit the "to do" here.

File: en/orm/database-basics.rst [hunk 2]
```diff
@@ -1,9 +1,100 @@
...
+the lower-level database access API. If you want instead to learn more about the
+full ORM, you can read the :doc:`/orm/query-builder` and :doc:`/orm/table-objects`
+sections.
+
+The easiest way to create a database connection is using a ``DSN`` string::
```
Comment (ravage84) [inline en/orm/database-basics.rst]: "a" becomes "an", isn't it?

Comment (lorenzo) [inline en/orm/database-basics.rst]: I don't think so, but I really cannot answer that for sure.

Comment (markstory) [inline en/orm/database-basics.rst]: No 'a' sounds right to me.

**Comment by markstory** [comment]
Looks great to me :+1:

**Comment by lorenzo** [comment]
I'll merge now, hope that is ok with @antograssiot :)

**Pull request merged**